benchmarks/index.md

```diff
@@ -50,22 +50,22 @@ with complex geometries (e.g., dendritic growth) and topology changes,
 such as particles merging or splitting, is challenging. The other
 mesoscale modeling approach uses diffuse (finite width)
 interfaces. This method can easily track complex interface geometries
-and topological changes such as particles merging. This benefit is not
+and topological changes such as particle merging. This benefit is not
```

Comment:
I think "particles" is more grammatically correct, but I could be mistaken, or there may be a term-of-art involved?

Reply:
I've always seen it written as I changed it, but I am not enough of a grammarian to look up the rule quickly.  I will reword it to avoid entirely.